src/clj/puppetlabs/puppetserver/certificate_authority.clj

```diff
@@ -1289,10 +1296,9 @@
   [{:keys [puppetserver jruby-puppet certificate-authority]}]
   (-> (select-keys puppetserver (keys CaSettings))
       (merge (select-keys certificate-authority (keys CaSettings)))
+      (assoc :auto-renewal-cert-ttl (get-auto-renewal-cert-ttl certificate-authority))
       (initialize-ca-config)
       (assoc :ruby-load-path (:ruby-load-path jruby-puppet)
-             :allow-auto-renewal (:allow-auto-renewal certificate-authority)
```

Comment:
This line shouldn't have been removed, and is a cause of new test failures.

Reply:
This will set the allow-auto-renewal value only from the certificate-authority, right? It won't honor the default value.

Reply:
Hmm.  I think this was in the old code?